content/docs/command-reference/metrics/add.md

```diff
@@ -18,42 +18,36 @@ defines the given `path` as an <abbr>output</abbr>, marking `path` as a metric
 file to track.
 
 Note that outputs can also be marked as metrics via the `-m` or `-M` options of
-`dvc run`.
+`dvc run`. We recommend using `-M` option to keep metrics in Git history.
 
-While any text file can be tracked as a metric file, we recommend using TSV,
-CSV, or JSON formats. DVC provides a way to parse those formats to get to a
-specific value, if the file contains multiple metrics. See the
-[options](#options) below and `dvc metrics show` for more info.
+While any text file can be tracked as a metric file, we recommend using JSON
+formats. DVC provides a way to parse this formats to get to a specific value, if
+the file contains multiple metrics. See the [options](#options) below and
+`dvc metrics diff` for more info.
```

Comment:
```suggestion
While any text file can be tracked as a metric file, we recommend using JSON
format. DVC provides a way to parse this format to get to a specific value, if
the file contains multiple metric values. See the [options](#options) below,
`dvc metrics show`, and `dvc metrics diff` for more info.
```
Notice I added `dvc metrics show` to the list which I understand is the only subcommand we're sure should keep the `---xpath` option. Is this correct?